src/adapters/IDB.js

```diff
@@ -238,4 +235,32 @@ export default class IDB extends BaseAdapter {
       });
     });
   }
+
+  /**
+   * In FakeIndexedDB, symbols are exposed using ``FDB`` prefixes in names.
+   * This piece of code will register them with the same names as native API,
+   * only if indexedDB is not already available.
+   *
+   * @param {Object} obj The object to check and atttach indexedDB symbols to.
+   */
+  static attachFakeIDBSymbolsTo(obj) {
```

Comment:
Now I think about it, the fake IDB stuff is only needed for test code ran in a nodejs environment, so we should probably move this to `test_utils.js` and call that function at test startup time.